incognia_test.go

```diff
@@ -204,6 +210,12 @@ var (
 		AccountID:      "account-id",
 		ExternalID:     "external-id",
 	}
+	loginFixtureWithEval = &Login{
```

Comment:
Doesn't `eval: true` have the same effect as `eval: nil`? If so, I think we should have a fixture for the other test case this is not covering (`loginFixtureWithoutEval`)